tests/gh-actions/katib_test.yaml

```diff
@@ -0,0 +1,78 @@
+---
```

Comment:
only nit I have would be to put the files/yamls we'll need for these tests into a distinct folder and not have everything flat under `tests/gh-actions`. Let's do `tests/gh-actions/kf-objects` WDYT?

Reply:
Yeap, I agree. I'll add a commit to arrange the folders like that.